dptb/postprocess/bandstructure/band.py

```diff
@@ -290,7 +255,7 @@ def estimate_E_fermi(cls, eigenvalues: np.array, total_electrons: int, spindeg:
         EF=(sorteigs[numek] + sorteigs[numek-1])/2
 
         return EF
-        
+
 
     def band_plot(
```

Comment:
the plot function should be able to use the inputed E_fermi to plot that band, but now it prioritize the estimated E_fermi.